backend/subroker/models.py

```diff
@@ -1,3 +1,124 @@
 from django.db import models
+from django.contrib.auth.models import User
 
 # Create your models here.
+
+class Ott(models.Model):
+    ott_type = (
+        ('Watcha', 'Watcha'), 
+        ('Netflix', 'Netflix'), 
+        ('Tving', 'Tving'), 
+        ('Youtube', 'Youtube'), 
+        ('Disney', 'Disney'), 
+        ('CoupangPlay', 'CoupangPlay'), 
+        ('Wavve', 'Wavve')
+        )
+    ott = models.CharField(choices=ott_type, max_length=15)
+    membership_type = (
+        ('Basic', 'Basic'), 
+        ('Standard', 'Standard'), 
+        ('Premium', 'Premium')
+        )
```

Comment:
maybe memberships vary platform by platform, is it ok to make only basic, standard, premium?